tro:  
  ros__parameters:
    splinesStride: 10
    Tmiddle: 0.05
    firstSpId: 0
    v0: 6.0

    Paths:
      Optimized: data/x_opt.csv
      Problem: data/problem.csv
      MiddlePoints: midline_points.csv
      FreeSpace: data/freeSpace.csv
      Save: data/